includes/class-newspack-listings-block-patterns.php

```diff
@@ -146,7 +132,33 @@ public static function register_block_patterns() {
+				],
+			],
+			'real_estate_2' => [
+				'post_types' => [ Core::NEWSPACK_LISTINGS_POST_TYPES['marketplace'] ],
```

Comment:
Strange, this was working for me last week but no longer. I find that registering the patterns on `init` (instead of `admin_init`) seem to fix it for me, what about you? d27b19a

Reply:
It does!